lib/yaml_extraction_prologue.sh

```diff
@@ -18,7 +18,7 @@ function extract_encrypted_variables() {
             (shyaml -q keys-0 <<<"${PLUGINS}" || true) |
             while IFS='' read -r -d '' PLUGIN_NAME; do
                 # Skip plugins that are not named `cryptic`
-                if [[ "${PLUGIN_NAME}" != staticfloat/cryptic* ]]; then
+                if [[ "${PLUGIN_NAME}" != staticfloat/cryptic* ]] && [[ "${PLUGIN_NAME}" != JuliaCI/cryptic* ]]; then
```

Comment:
How was this working before, given we moved this repo to JuliaCI?

Reply:
I guess it's just what we call it, and github is forwarding from the old url https://github.com/staticfloat/cryptic-buildkite-plugin